packages/mdc-tab-indicator/_mixins.scss

```diff
@@ -29,14 +29,13 @@
 
 @mixin mdc-tab-indicator-underline-color($color) {
   > .mdc-tab-indicator__content--underline {
-    @include mdc-theme-prop(background-color, $color);
```

Comment:
I thought this was still required? (per convo from yesterday)

Reply:
I updated the height mixin to set the border-top-width instead of the actual height prop.

https://github.com/material-components/material-components-web/blob/1f1bf2c8b2afb051c431e200c2451e4d7362f3a9/packages/mdc-tab-indicator/_mixins.scss#L36-L40

Reply:
Ok that makes sense. This works for me.